test/k6/src/api/instances.js

```diff
@@ -5,40 +5,52 @@ import { stopIterationOnFail } from "../errorhandler.js";
 
 /**
  * Api call to Storage:Instances to create an app instance and returns response
- * @param {*} token token value to be sent in apiHelper for authentication
- * @param {*} partyId party id of the user to whom instance is to be created
- * @param {*} org short name of ap owner
- * @param {*} app app name
- * @param {JSON} serializedInstance instance json metadata sent in request body
- * @returns {JSON} Json object including response apiHelperss, body, timings
+ * @param {Object} instanciationOptions Object containing the parameters for the API call.
```

Comment:
spelling error `instanciation` should be `instantiation`

Reply:
What a silly mistake! I should have been more careful! My bad! Fixed now! 👍 Marking is resolved. 🙂